src/legacy/components/AppState.ts

```diff
@@ -32,7 +32,9 @@ export class RPCConfig {
   password = ''
   url = ''
 }
-export class Info {}
+export class Info {
+  [x: string]: any
```

Comment:
Please don't use any type.

Right now it's not completely clear how currency name will be dynamically retrieved, but `class Info` with any properties, seems to be a bad idea.